karaf/configs/com.cognifide.aet.runner.RunnerConfiguration.cfg

```diff
@@ -18,6 +18,6 @@
 
 ft=120
 mttl=300
-urlPackageSize=1
-maxMessagesInCollectorQueue=10
+urlPackageSize=5
+maxMessagesInCollectorQueue=20
```

Comment:
This should remain `6 - 10` here. It defines the maximum number of collection tasks waiting in the queue. It should be at least the number of collectors in the system (if it is less some of collectors will be never used) but not to big (as it will occur with timeouts on collection taks). See more in https://github.com/Cognifide/aet/issues/361 about this value.